enhancements/cluster-logging/cluster-logging-log-forwarding.md

```diff
@@ -99,259 +116,211 @@ This is often required for industries such as the US public sector, healthcare o
+   - inputs: [ Infrastructure, Application, Audit ]
+     outputs: [ SecureRemote, Default ]
 ```
 
-The generated collector configuration is something like the following. **Note:** the source definitions from prior releases remain unchanged:
+#### As a cluster administrator, I want to use a local syslog instance only, with no elasticsearch costs.
 
 ```
-# source definitions
-<source>
-  @type tail
-  @label @CONCAT
-  <label @CONCAT>
-    <filter kubernetes.**>
-      @type concat
-      key log
-      partial_key logtag
-      partial_value P
-      separator \'\'
-    </filter>
-    <match kubernetes.**>
-      @type relabel
-      @label @INGRESS
-    </match>
-  </label>
-</source>
- 
-# INGRESS
-<label @INGRESS>
-  ... #current @INGRESS config
-
-    <match output_ops_tag journal.** system.var.log** mux.ops audit.log**   %OCP_FLUENTD_TAGS%>
-      @type relabel
-      @label @_LOGS_INFRA
-    </match>
-    <match kubernetes.**>
-      @type relabel
-      @label @_LOGS_APP
-    </match>
-</label>
-
-<label @_LOGS_APP>
-  <match **>
-     @type copy
-     <store>
-        @type relabel
-        @label @CONTAINER_LOGS
-     </store>
-  </match>
-</label>
-
-<label @_LOGS_INFRA>
-  <match **>
-     @type copy
-     <store>
-        @type relabel
-        @label @INFRA_LOGS
-     </store>
-  </match>
-</label>
-
-# match outputs are generated based on defined pipeline endpoints
-<label @CONTAINER_LOGS>
-  <match **>
-     @type copy
-     #store section for each endpoint type
-     <store>
-       @type relabel
-       @label @SECUREFORWARD_OFFCLUSTER
-     </store>
-     <store>
-       @type relabel
-       @label @ELASTICSEARCH
-     </store>
- </match>
-</label>
-
-<label @CONTAINER_LOGS>
-  <match **>
-     @type copy
-     #store section for each endpoint type
-     <store>
-       @type relabel
-       @label @ELASTICSEARCH_INSECURE
-     </store>
- </match>
-</label>
- 
-#specific pipeline endpoint
-<label @ELASTICSEARCH>
-  <match retry_elasticsearch>
-     @type copy
-     <store>
-       @type elasticsearch
-       @id elasticsearch
-       client_key /var/run/ocp-collector/secrets/elasticsearch/key
-       client_cert /var/run/ocp-collector/secrets/elasticsearch/cert
-       ca_file /var/run/ocp-collector/secrets/elasticsearch/cacert
-       <buffer>
-		@type file
-		path '/var/lib/fluentd/elasticsearch'
-       </buffer>
-     </store>
-  </match>
-  <match **>
-     @type copy
-     <store>
-       @type elasticsearch
-       @id elasticsearch
-       client_key /var/run/ocp-collector/secrets/elasticsearch/key
-       client_cert /var/run/ocp-collector/secrets/elasticsearch/cert
-       ca_file /var/run/ocp-collector/secrets/elasticsearch/cacert
-       retry_tag "retry_elasticsearch"
-     </store>
- </match>
-</label>
-<label @SECUREFORWARD_OFFCLUSTER>
-   ...
-</label>
+apiVersion: "logging.openshift.io/v1"
+kind: "ClusterLogForwarder"
+spec:
+  outputs:
+   - name: MyLogs
+     type: Syslog
+     syslog: {Facility: Local0}
```

Comment:
i think this should be:
```
type: Syslog
syslog:
  facility: Local0
```

or would `syslog` take in a string that we would have to then parse?

Reply:
Both forms are exactly equivalent to a YAML parser, but I'll normalize on your form since it's more common and I don't want to distract.

Reply:
- inputs: [Infrastructure, Application, Audit]